system/emergency_handler/src/emergency_handler/emergency_handler_core.cpp

```diff
@@ -407,15 +405,14 @@ autoware_system_msgs::msg::HazardStatus EmergencyHandler::judgeHazardStatus()
           "heartbeat_is_state_timeout is timeout"));
     }
 
-    if (is_state_timeout_->is_timeout) {
+    if (is_state_timeout_->is_timeout && !is_in_timeout_ignore_state) {
```

Comment:
I think we should ignore only heartbeat timeouts. This is not a heartbeat.
So please rename `is_in_timeout_ignore_state` to `is_in_heartbeat_timeout_ignore_state` and don't ignore is_state_timeout_->is_timeout.

Reply:
Dealt with it @ 7b2aee5